src/domain/buildingBlocks/Pluto.ts

```diff
@@ -66,16 +66,6 @@ export interface Pluto {
    */
   storeMessages(messages: Array<Message>): Promise<void>;
 
-  /**
-   * Store a list of private keys with its metadata and a reference to the DID it belongs to.
-   */
-  storePrivateKeys(
-    privateKey: PrivateKey,
-    did: DID,
-    keyPathIndex: number,
-    metaId: string | null
-  ): Promise<void>;
```

Comment:
not explicitly used by us in code.
should be used by proxy in `storePrismDID` and `storePeerDID`.